files/var/lib/cloud/scripts/per-instance/01-run-appwrite.sh

```diff
@@ -2,9 +2,14 @@
 
 cd /root
 
-docker pull appwrite/appwrite:1.3
+appversion=$(curl -s https://api.github.com/repos/appwrite/appwrite/releases/latest | grep -oP '"tag_name": "\K.*?(?=")')
+
+echo "==============================================="
+echo "=       Installing appwrite $appversion       ="
+echo "==============================================="
+
 
 docker run --rm \
     --volume /var/run/docker.sock:/var/run/docker.sock \
     --volume "$(pwd)"/appwrite:/usr/src/code/appwrite:rw \
-    appwrite/appwrite:1.3 sh -c "install --httpPort=80 --httpsPort=443 --interactive=N"
\ No newline at end of file
+    appwrite/appwrite:$appversion sh -c "install --httpPort=80 --httpsPort=443 --interactive=N"
```

Comment:
If we always want to get the latest, we could just use `appwrite/appwrite:latest`, right?
```suggestion
    appwrite/appwrite:latest sh -c "install --httpPort=80 --httpsPort=443 --interactive=N"
```

Reply:
I think that will also create the `docker-compose.yml` file with `apwrite:latest` (I'm not sure) so if docker-compose has latest, if they always run `docker pull latest` there might be confusion as to which version they are at, which version they will be upgrading to, do they need to migrate and all?

Reply:
That version value is pulled from an env var in the docker image so it should be okay: 

<img width="867" alt="image" src="https://github.com/appwrite/integration-for-digitalocean/assets/1477010/f4d873f4-4f63-47da-bd85-411ed4404103">

Reply:
Still want to keep this, so even if there is error with updating latest image, we should be good.